docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 1
+sidebar_label: "Configuration"
+id: daemon-config
+title: Configuration
+description: How to configure the Kwil Daemon
+slug: /daemon/config/
+---
+
+Kwild application reads its config from the config.toml located in the home_dir/abci/config/config.toml. This include both the Kwildb and the chain configuration.
+
+## Kwild Application config
+### Basic config
```

Comment:
I know that this lives within the `app` section of the config.  Maybe having a title that explains this, as well as explaining what the titles mean in the intro would make sense.  For example:

(top of the page)

... Each section outlines a section of the config.  The general structure is:

## <section_1>
### <subsection_1>
- `CONFIG_NAME`: description
### <subsection_2>
## <section_2>

This is the structure that these configs must appear in the config file.  These values can also be set using flags via dot notation.  In the above example, `CONFIG_VAL` can be referenced by: `--<section_1>.<subsection_1>.CONFIG_VAL=<your_value>`

Something along these lines might help give clarity.